SS_readcontrol_330.tpl

```diff
@@ -3071,15 +3073,15 @@
     {
       if (Q_setup_check(f) > 0)
       {
-        warnstream << f << " Q setup error; no survey obs but Q setup was read ";
+        warnstream << " Q setup error; no survey obs for fleet " << f << " but Q setup was read ";
         write_message (FATAL, 0); // EXIT!
       }
     }
   }
-  
-  echoinput << "q setup " << endl
-            << Q_setup << endl;
-  echoinput << "q setup parms " << endl
+
+  echoinput << "Q_Npar (number of long parameter lines): " << Q_Npar << endl
+            << endl << "Q setup parameter index " << endl
+            << "column have index of first parameter for:  1=base q with link;  2=extrastd; 3=env; 4=block/trend; 5=dev;"
```

Comment:
Should there be " << endl" here? Also, same question about ";" at the end of lines here.